.readthedocs-requirements.txt

```diff
@@ -4,3 +4,5 @@ numpy
 matplotlib
 pillow
 sphinx-gallery
+pandas
```

Comment:
Before I forget tt would be to mention in the README.rst that pandas is needed to run some examples.

Reply:
The notes about building the docs already mention that some dependencies are required and gives the command to install them: https://github.com/joblib/joblib#building-the-docs.

Reply:
I am sure you are aware that joblib users may want to run the examples without building the doc. They may even be completely unaware that both are related. Add a sentence after the optional numpy dependency and/or look at how scikit-learn does it.

Reply:
I have seen this has been addressed, sorry for the noise.